mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -232,7 +232,7 @@ properties:
                         description:
                           Hostname of the machine. VM's name will be used if
                           this field is empty.
-                        required: true
+                        default_from_api: true
```

Comment:
May I ask what's the reason of the change?

Reply:
Thanks for asking. This is a fix on the initial resource schema. The hostname should be optional and defaulted on server side. It is only required to provide ip address for static ip config. @hao-nan-li

Reply:
Thanks for the explanation!